packages/react-inbox/src/components/Inbox/index.tsx

```diff
@@ -245,6 +245,10 @@ const Inbox: React.FunctionComponent<InboxProps> = (props) => {
 
   const isMobile = windowSize?.width ? windowSize?.width <= 480 : false;
 
+  const BellWrapper = () => {
+    return props.renderBell ? props.renderBell({}) ?? bell : bell;
```

Comment:
we need to pass all the same props that get assigned to the current bell to this renderBell function